### ISSUE TITLE
Error: Unhandled rejection TypeError: host.getCanonicalFileName is not a function

### ISSUE THREAD

**Issue by OrKoN** [charto/docts#1]
Hi,

I seem to be getting the following error on my project. Could you suggest something?
Stack trace:

```
readts/dist/Parser.js:31:31
docts/dist/DocBuilder.js:216:42
```

Thanks in advance,
Alex

**Comment by sallar** [comment]
having the same issue

**Comment by sallar** [comment]
The error is coming from this section in `readts` module:

```js
            if (!nameFilter ||
                !extension ||
                nameFilter(ts.getOwnEmitOutputFilePath(source, this.program, extension))) {
                this.parseSource(source);
            }
```
The `ts.getOwnEmitOutputFilePath` function is throwing an error.

**Comment by jjrv** [comment]
This should be fixed now in version 0.2.0.
